src/test/java/com/jcabi/github/RtStatusesTest.java

```diff
@@ -0,0 +1,110 @@
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.github;
+
+import com.jcabi.github.mock.MkGithub;
+import com.jcabi.http.Request;
+import com.jcabi.http.mock.MkAnswer;
+import com.jcabi.http.mock.MkContainer;
+import com.jcabi.http.mock.MkGrizzlyContainer;
+import com.jcabi.http.request.ApacheRequest;
+import java.net.HttpURLConnection;
+import javax.json.Json;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Testcase for RtCommits.
+ *
+ * @author Marcin Cylke (marcin.cylke+github@gmail.com)
+ * @version $Id$
+ * @checkstyle MultipleStringLiterals (500 lines)
+ */
+public class RtStatusesTest {
+
+    /**
+     * Tests creating a Commit.
+     *
+     * @throws Exception when an error occurs
+     */
+    @Test
+    public final void createsStatus() throws Exception {
+        final String sha = "0abcd89jcabitest";
+        final MkContainer container = new MkGrizzlyContainer().next(
+                new MkAnswer.Simple(
+                    HttpURLConnection.HTTP_CREATED,
+                    Json.createObjectBuilder().add("state", "failure")
+                        .add("target_url", "https://ci.example.com/1000/output")
+                        .add("description", "Build has completed successfully")
+                        .add("context", "continuous-integration/jenkins")
+                        .build().toString()
+            )
+        ).start();
+        final Request req = new ApacheRequest(container.home());
+        final Statuses statuses = new RtStatuses(
+                req, new RtCommit(req, repo(), sha)
+        );
+        try {
+            final Status status = statuses.create(
+                    new RtStatus(
+                            Status.State.failure, "http://example.com",
+                            "description", "ctx"
+                )
+            );
+            MatcherAssert.assertThat(
+                    status,
+                Matchers.instanceOf(Status.class)
+            );
+            MatcherAssert.assertThat(
+                container.take().method(),
+                Matchers.equalTo(Request.POST)
+            );
+            MatcherAssert.assertThat(
+                status.state().name(),
+                Matchers.equalTo(Status.State.failure.name())
```

Comment:
equals works fine for enums.
We can use ```Matchers.is()``` or ```Matchers.equalTo()``` (which is the same) for enums, not their names